### PR TITLE
02567 Enable jasperdb Logging

### PR DESCRIPTION
**Pull request by rbair23**

### CODE REVIEW

**Review by tinker-michaelj — APPROVED** [review]
LGTM

File: hedera-node/configuration/mainnet/log4j2.xml [hunk 1]
```diff
@@ -132,6 +132,8 @@
         <MarkerFilter marker="MERKLE_HASHING"         onMatch="DENY"   onMismatch="NEUTRAL"/>
         <MarkerFilter marker="MERKLE_GENERATION"      onMatch="DENY"   onMismatch="NEUTRAL"/>
         <MarkerFilter marker="MERKLE_LOCKS"           onMatch="DENY"   onMismatch="NEUTRAL"/>
+        <MarkerFilter marker="JASPER_DB"              onMatch="ACCEPT" onMismatch="NEUTRAL"/>
```
Comment (superboo) [inline hedera-node/configuration/mainnet/log4j2.xml:135]: I'm assuming that the marker JASPER_DB is not used frequently and only for errors?

**Review by superboo — APPROVED** [review]
LGTM

**Comment by tinker-michaelj** [comment]
Was done in https://github.com/hashgraph/hedera-services/pull/2608

**Pull request closed without merging**